packages/desktop-gui/src/lib/browser-model.js

```diff
@@ -25,6 +26,14 @@ export default class Browser {
   }
 
   @computed get icon () {
-    return 'chrome'
+    if (this.family === 'electron') {
+      return 'chrome'
+    }
+
+    return _
```

Comment:
```js
return _.get({
  chrome: 'chrome',
  firefox: 'firefox'
}, this.family, 'globe') 
```
https://lodash.com/docs/4.17.15#get

Reply:
lol, everybody has an opinion here 🙃 I agree this is simpler than @brian-mann suggestion.

Reply:
turns out this is implemented in another place too: https://github.com/cypress-io/cypress/blob/6405fa66c688c03bb1188240400db61bc5af42f0/packages/desktop-gui/src/lib/utils.js#L13-L18 https://github.com/cypress-io/cypress/blob/6405fa66c688c03bb1188240400db61bc5af42f0/packages/desktop-gui/src/lib/utils.js#L28-L32